community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/main.tf

```diff
@@ -49,4 +49,6 @@ locals {
     reserved        = var.reserved
     network_storage = var.network_storage
   }
+
+  tpu_core_count = tonumber(regex("-(.*)", var.node_type)[0])
```

Comment:
you can either use node_type or accelerator config to provide TPU accelerator configuration and get TPU core count. https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/variables.tf#L56

Reply:
I see, it appears node_type is a required [variable](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/variables.tf#L51). So curious about the use of accelerator config object in these instances, it doesn't seem like node_type could be null for [this condition](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/f9f92565a7a4133e0c7b31cd55e56cb0728dbb2c/community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/outputs.tf#L20) to be activated.

Added additional logic for accelerator config types.